### PR TITLE
1248/mobile scroll issue

### PR DESCRIPTION
**Pull request by ColinBuyck**
## Issue

- Addresses #1248 

## Description

The previous issue was that the filter menu for the listings page on mobile would only scroll with two fingers and would occasionally scroll the listings behind it. After spending considerable time working with allowTouchMove, I found that the package itself was not working effectively on mobile. I replaced it with [react-remove-scroll.](https://github.com/theKashey/react-remove-scroll) which is a much simpler and more consistent solution.

## How Can This Be Tested/Reviewed?

Go to the listings page on Detroit's public site on your mobile device, and interact with the filtering functionality.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by ColinBuyck** [comment]
Note: Working on this issue highlighted the fact that the submit button is covered by some mobile browser's toolbar (Safari and Chrome have this issue; Google App does not). Should I address this in this PR or separate it out?

**Review by YazeedLoonat — COMMENTED** [review]
this looks good to me code-wise but I don't have an IOS phone so I can't test the end results

**Review by seanmalbert — CHANGES_REQUESTED** [review]
Not sure if this happens in the real-world, but testing in browserstack with an iPhone 13 in both safari and chrome, when I scroll down to the bottom, it does this weird staggered jump down the screen until the done and clear buttons are no longer visible. I can scroll back up and then down to see the buttons, but it happens each time, so I affectively can't submit or clear the filters.

If someone with e real iPhone can confirm this doesn't happen, then I'll approve.

Tested with the preview https://deploy-preview-1279--detroit-public-dev.netlify.app/

**Comment by ColinBuyck** [comment]
@seanmalbert I think this is the issue referenced above. It was present with the previous package, but I can work on it as part of this issue if that's preferred!

**Review by seanmalbert — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "clean": "rm -rf backend/core/dist/ backend/core/node_modules/ node_modules/ sites/partners/.next/ sites/partners/node_modules/ sites/public/.next/ sites/public/node_modules/ ui-components/node_modules/"
   },
   "dependencies": {
+    "react-remove-scroll": "^2.5.3",
```
Comment (jaredcwhite) [inline package.json:57]: Do we want the dependency added here, rather than in `ui-components`?

**Review by jaredcwhite — COMMENTED** [review]
@ColinBuyck @seanmalbert I know this got merged, but I had one question about where best to place the dependency.

**Pull request merged**